CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+### Changed
+
+- Bump `psycopg` to v3
+- **breaking** Using `sqlalchemy` to request Postgres databases. SQL array type parameters are no longer converted into `tuples` so queries must use the `ANY` operator instead of `IN`.
```

Comment:
```suggestion
- **breaking** The Postgres connector's driver switched from `psycopg2` to `psycopg3`. As a consequence, array parameters are no longer interpolated as tuples in strings, meaning that in order to use query parameters, the `ANY()` function should be used instead of the `IN` operator: https://www.psycopg.org/psycopg3/docs/basic/from_pg2.html#you-cannot-use-in-s-with-a-tuple
* The Postgres connector now uses `psycopg` via `sqlalchemy` rather than directly
```